exercises/tournament/src/test/java/TournamentTest.java

```diff
@@ -0,0 +1,158 @@
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class TournamentTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
```

Comment:
Why do you need this? You only need this if you're expecting an exception to be thrown in one of your tests and from what I can tell you're not :)